src/Native/pkg/runtime.native.System/runtime.native.System.builds

```diff
@@ -11,10 +11,14 @@
       <OSGroup>debian.8</OSGroup>
       <Platform>amd64</Platform>
     </Project>
-    <Project Include="ubuntu\runtime.native.System.pkgproj">
+    <Project Include="ubuntu\14.04\runtime.native.System.pkgproj">
```

Comment:
Maybe this a question for @ericstj @chcosta or @jhendrixMSFT but long term will it be possible to reduce the ceremony here?  Could we not share the same pkgproj file for all the builds and just use properties like `$(OSGroup)` (and maybe some additional new ones) so we didn't have to duplicate this file per distro/version?